attribute_set/models/ir_model.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV (http://www.acsone.eu).
+# @author Cédric PIGEON <cedric.pigeon@acsone.eu>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class IrModelFields(models.Model):
+    _inherit = 'ir.model.fields'
+
+    @api.model
+    def _instanciate_attrs(self, field_data, partial):
+        attrs = super(IrModelFields, self)._instanciate_attrs(field_data,
+                                                              partial)
+        name = field_data.get('name')
+        if name.startswith('x_'):
```

Comment:
I think this part doesn't work in this case:
- You have existing custom fields (starting with `x_`);
- You install this module;
=> That raise an exception because the model attribute.attribute doesn't exist (yet).

Because this function is called during field initialisation and the table `attribute_attribute` is not created yet.
That happens only if you do the first install of the module with this PR; If the module is already installed without this PR and then upgraded (including this PR) the `attribute_attribute` table already exists so no issue.

I think the query should be encapsulated into that kind of `if`:
`if 'attribute.attribute' in self.env`:
 `...`